### PR TITLE
fix: [CDS-48410]: remove readonly configure options for services

### PR DESCRIPTION
**Pull request by AshwinBhatkal-Harness**
### Summary

Removed configure options for **service/services**
Removed execution time support for **service/services**.
Removed expression support for **services**

#### Screenshots

image

image

image

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vikrant-harness — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityInputStep.tsx [hunk 1]
```diff
@@ -269,23 +274,6 @@ export function DeployServiceEntityInputStep({
               />
             )
           ) : null}
-          {getMultiTypeFromValue(get(formik?.values, `${localPathPrefix}serviceRef`)) ===
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityInputStep.tsx:272]: configure options removed from service in input view

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityInputStep.tsx [hunk 2]
```diff
@@ -300,7 +288,7 @@ export function DeployServiceEntityInputStep({
               isNewConnectorLabelVisible={false}
               multiTypeProps={{
                 expressions,
-                allowableTypes
+                allowableTypes: allowableTypesWithoutExpressionExecution
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityInputStep.tsx:291]: execution and expression support removed from service in input view for single service

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx [hunk 3]
```diff
@@ -524,10 +524,8 @@ export default function DeployServiceEntityWidget({
                                 }}
                                 multiTypeProps={{
                                   width: 300,
-                                  expressions,
-                                  allowableTypes
+                                  allowableTypes: getAllowableTypesWithoutExpression(allowableTypes)
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx:527]: expression support removed from multi service

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx [hunk 4]
```diff
@@ -546,7 +544,7 @@ export default function DeployServiceEntityWidget({
                                 width={300}
                                 multiTypeProps={{
                                   expressions,
-                                  allowableTypes,
+                                  allowableTypes: getAllowableTypesWithoutExpression(allowableTypes),
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx:547]: expression support removed from multi service

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx [hunk 5]
```diff
@@ -599,21 +597,6 @@ export default function DeployServiceEntityWidget({
                                 selectItems={selectOptions}
                               />
                             )}
-                            {getMultiTypeFromValue(formik?.values.service) === MultiTypeInputType.RUNTIME && (
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx:602]: configure options removed from service in pipeline

**Review by vikrant-harness — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**